insta_reels_publishing_api_sample/.env.template

```diff
@@ -4,3 +4,11 @@ API_SECRET=YOUR_APP_SECRET
 REDIRECT_URI=https://localhost:8000/callback
 SESSION_SECRET=RANDOM_SESSION_SECRET_STRING
 PORT=8000
+
+# Optional config setting to change the Graph API version.
+# Leave blank or commented to default to the app's default version
+# GRAPH_API_VERSION_PATH=v18
```

Comment:
Can we change this to `v18.0`?

<img width="88" alt="image" src="https://github.com/fbsamples/reels_publishing_apis/assets/9583991/fce2664e-39b3-47cc-9391-954691a8074a">

Reply:
Changed. I had missed it, thanks!